pkg/virtctl/imageupload/imageupload.go

```diff
@@ -175,11 +176,20 @@ func (c *command) run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	if uploadProxyURL == "" {
+		uploadProxyURL, err = getUploadProxyURL(virtClient.CdiClient())
+		if err != nil {
+			return err
+		}
+		if uploadProxyURL == "" {
```

Comment:
I have see  the pattern to return a tripple too, something like ` func() (url string, exists bool, err error)`